scripts/mdpow-pow

```diff
@@ -208,7 +208,7 @@ if __name__ == "__main__":
                       help="force rereading all data [%default]")
     parser.add_option('--noSI', dest='SI',
```

Comment:
Rename as `--no-SI` so that when we use argparse bool options, it has the same name.

Reply:
To make things clear, also add 
```python
  parser.add_option('--SI', dest='SI', action="store_true", ...)
```
as the on switch. Set the default for `SI`  to `True`.